.gitignore

```diff
@@ -102,3 +102,6 @@ dist
 
 # TernJS port file
 .tern-port
+
```

Comment:
```suggestion
.prettierrc
```

Reply:
.prettierrc is not supposed to be in gitignore... but thank you for the suggestion